src/utils/theme.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import _ from 'lodash/fp'
+
+let ThemeContext = React.createContext()
+
+export let ThemeProvider = ({ children, theme }) => (
```

Comment:
This probably doesn’t add much value over just exporting ThemeContext.Provider directly. I’m fine with calling the prop `value` instead of `theme`

Reply:
For what it's worth, I left this in for two reasons:
- parity with ThemeConsumer (which is kind of an extra anyway)
- it matches the theme APIs for styled-components and emotion

Both of those are pretty minor, so no objection here 🙂

Reply:
The advantage of not doing is that it's react context, nothing special about our API. Plus, its less code